package/MDAnalysis/topology/PDBParser.py

```diff
@@ -303,6 +308,23 @@ def _parseatoms(self):
         masses = guess_masses(atomtypes)
         attrs.append(Masses(masses, guessed=True))
 
+        # Guessed attributes
+        # Need to pull elements from Atom names
+        # Similar to the check for atomtypes function
+        if any(elements):
```

Comment:
The logic was if any of the element identifier exists, then get element information, else guess based on atom names, and report individual missing elements. However, if the entire row is missing, then the element identifier does not exist, and hence, we would be guessing completely based on the atom names. The subset of missing element identifiers can vary, hence I wanted to prompt a warning only in the cases when it was missing, hence I did a if any instead of a if not all. Because only specific elements are being guessed instead of the entire atom set in the PDB. In your opinion should I change this logic?